### DISCUSSION TITLE
Add the option to draw a border around the lsp parameter hint popup window

### DISCUSSION THREAD

**Discussion by lominoss-git** [helix-editor/helix#7308] [Ideas]
Just like the one the file picker and the command description have.
Here is how it would look (neovim screenshot)
![beauty](https://github.com/helix-editor/helix/assets/79030093/cd3ee6b4-9b06-4373-944e-57fef5e6b64f)

**Comment by gabydd** [comment]
I think this should be covered by https://github.com/helix-editor/helix/pull/4313
